### PR TITLE
DRAFT: VOPs draft for feedback.

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: RPIPs/RPIP-vop.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+* pDAO Budget Split
+* Rocket Pool Inflation Allocation
+* GMC Internal Decisions
+* IMC Internal Decisions
+
+## Security Considerations
+
+There are not many security considerations for this RPIP, as it is non-technical. Modifications to the logic of any existing processes are limited to separating those processes from the main RPIPs framework. 
+
+The possibility exists that the attached PR and modifications to existing RPIPs may unintentionally modify the logic or rules governing existing processes. This is mitigated by explicitly giving RPIP Editors the power to restore the intended logic or rules without a further vote.
```
Comment (patrick-j-govalpha) [inline RPIPs/RPIP-vop.md:360]: Minor: more concise
```suggestion
There exists a risk that the proposed changes could inadvertently alter the logic or rules of existing processes. This is mitigated by explicitly giving RPIP Editors the power to restore the intended logic or rules without a further vote.
```

**Review by patrick-j-govalpha — COMMENTED** [review]
I think this is in a good place tbh, I've left some comments, feel free to disregard any you think are pointless

File: RPIPs/RPIP-vop.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+
+## Specification
+
+### VOPs: What and When?
+This section describes what a VOP is, and when to use the VOP framework.
+
+* A Voted Operational Process (VOP) is a process involving a vote through which an operational decision is made.
+  * Some operational decisions within the Rocket Pool DAOs MAY NOT require a vote, and should not be included under the VOP framework.
+* A VOP MUST NOT be defined to cover decisions that are improvements to the Rocket Pool protocol, Rocket Pool DAOs, or anything that should otherwise require the use of the RPIP process.
+* The dividing line between which decisions should use the RPIP process versus the VOP framework may be unavoidably blurry. The following general recommendations are made:
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply RPIPs/RPIP-vop.md:47]: Prose suggested that too. I'm not sure I quite get it. Will bring it up at the meeting.

File: RPIPs/RPIP-vop.md [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+  * A VOP Record MUST include the date the VOP concluded.
+  * A VOP Record MUST include an appropriate URL link indicating the outcome of the VOP.
+  * A VOP Record MUST include a brief, human-readable summary of the outcome.
+
+---
+
+### VOP Usage - Overview
+> A Living informational RPIP that lists the currently defined VOPs and the RPIPs in which they are defined.
+
+* The VOP Overview is a summary of defined VOPs within Rocket Pool.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply RPIPs/RPIP-vop.md:125]: Hadn't considered expired VOPs. Good point. Will cover that somewhere.

File: RPIPs/RPIP-vop.md [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+
+The intention is for overviews such as this to exist in a single informational RPIP that lists all defined VOPs that exist in the Rocket Pool DAOs. These descriptors will allow newcomers to the Rocket Pool community to easily identify common processes that take place at a high-level, and learn more about any they are interested in.
+
+---
+
+## Rationale
+
+### General Structure
+
+The general structure proposed here isn't anything new or ground-breaking. The value is in better categorizing decision-making such that decisions with very different characteristics don't _look the same_ to those needing to make them. This structure is the most lightweight possible to allow some form of categorization. Hierarchical structures are an option with more complexity, but I don't feel there is an obvious need to pay that cost within Rocket Pool at this time. A compositional structure with tagging is another option, tags can represent a better user experience, but they do require more active and consistent maintenance, which I don't believe can be relied upon based on the current state of the Rocket Pool DAOs.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply RPIPs/RPIP-vop.md:223]: Will adjust, but not to suggestion as written, is also awkward.

File: RPIPs/RPIP-vop.md [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+
+### Overview
+
+Having an overview in some form is valuable to less-engaged participants, and those wanting to onboard into the DAO. Arguably this doesn't need to be in an Informational RPIP, other options are some official documentation hosted elsewhere, or to try to generate this dynamically based on the RPIP content. Dynamic generation has some advantages, though it can't replicate a hand-written description by the RPIP editors. I'm planning to experiment a bit with dynamic generation if this should pass. If I end up with something good, I may propose removing the informational RPIP in favor of a dynamic page on the portal. Ultimately though, adding an informational RPIP is just the most obvious and easiest immediate solution. 
+
+
+### Definitions
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply RPIPs/RPIP-vop.md:230]: It's rationale? Feels fine for this to be more informal, but can change if you and Prose feel strongly.

File: RPIPs/RPIP-vop.md [hunk 6]
```diff
@@ -0,0 +1,365 @@
...
+
+### Records
+
+Records in some form are valuable. Tables make the most sense, as they are fairly compact, and make good use of horizontal space once you have a reasonable amount of columns.
+
+## Backwards Compatibility
+
+If this RPIP is ratified, existing ratified RPIPs containing processes that would fall under the VOP framework will be modified to use the VOP framework. The explicit intention is to maintain the logic and rules of existing processes, and just change their presentation such that they integrate with the VOP framework.
+
+In case of unintentional changes to the logic of existing processes, or ambiguity introduced by the changes, RPIP Editors MAY modify the relevant RPIPs to _restore logical parity only_ without requiring a pDAO vote.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply RPIPs/RPIP-vop.md:248]: Will modify to somewhere in the middle.

**Pull request closed without merging**